macros/src/actix.rs

```diff
@@ -129,23 +149,37 @@ pub fn emit_v2_operation(attrs: TokenStream, input: TokenStream) -> TokenStream
                 let f = #wrapped_fn_call;
                 paperclip::actix::ResponseWrapper {
                     0: f,
-                    1: #unit_struct,
+                    1: #unit_struct #generics_call,
                 }
             }
         ))
         .expect("parsing wrapped block"),
     );
 
-    // panic!("{}",
+    let docs = extract_documentation(&item_ast.attrs);
+    let mut lines = docs.lines();
+    let summary = lines
+        .next()
+        .map(|line| quote!(Some(#line.to_string())))
+        .unwrap_or(quote!(None));
+    let description = lines.collect::<String>().trim().to_string();
+    let description = if !description.is_empty() {
```

Comment:
Let's also expect a newline between summary and description for the time being? (as it's consistent with Rust docs). Also, this will be our last resort eventually, because I really want to make use of the attribute macro to achieve this (like `#[api_v2_operation(summary = "...", description = "...")]`

Reply:
Neat point, pushed a fix